docs/en/integrations/data-ingestion/apache-spark/index.md

```diff
@@ -0,0 +1,306 @@
+---
+sidebar_label: Integrating Apache Spark with ClickHouse
+sidebar_position: 1
+slug: /en/integrations/apache-spark/
+description: Introduction to Apache Spark with ClickHouse
+---
+
+# Integrating Apache Spark with ClickHouse
+
+[Apache Spark](https://spark.apache.org/) Apache Spark™ is a multi-language engine for executing data engineering, data science, and machine learning on single-node machines or clusters.
+
+# Get the Library
+
+## Download the Library
```

Comment:
Do we need to add a list of supported types? Complex types might be tricky https://github.com/ClickHouse/clickhouse-java/issues/1042